packages/bitcore-node/src/providers/chain-state/internal/internal.ts

```diff
@@ -591,7 +603,7 @@ export class InternalStateProvider implements IChainStateService {
    *
    * @returns Array<string>
```

Comment:
Recommend removing jsdoc or changing to parity with TS return type:
`@returns {Promise<Array<string>>}`